### PR TITLE
instructorCourseDetailsPage: give a link to enroll students if there are no students #3966

### LINKED ISSUE #3966: instructorCourseDetailsPage: give a link to enroll students if there are no students
givealinktoenroll

We can add a link to the enrolment page in the status message

### PR DESCRIPTION
**Pull request by unyoungwax**
Fixes #3966

### CODE REVIEW

File: src/test/resources/pages/InstructorCourseDetailsEmptyCourse.html [hunk 1]
```diff
@@ -271,7 +271,11 @@
             </div>
             <br/>
             <div class="alert alert-warning" id="statusMessage">
-               There are no students in this course
+               There are no students in this course. Click
+               <a href="/page/instructorCourseEnrollPage?courseid=CCDetailsUiT.CourseWithoutStudents&user=CCDetailsUiT.instr">
+                  here
+               </a>
+               to enroll students.
```
Comment (wkurniawan07) [inline src/test/resources/pages/InstructorCourseDetailsEmptyCourse.html]: Just a quick question, do you fix this file manually?

**Comment by unyoungwax** [comment]
Yes

**Comment by wkurniawan07** [comment]
Ok, I was just wondering :p This looks good as long as dev green, well done :+1:

**Comment by damithc** [comment]
Reviewer needs to merge this as @unyoungwax doesn't have permission to merge. May be run through God mode before merging? If not, the change to html file may not match what God mode would have done, right?

**Comment by wkurniawan07** [comment]
@damithc actually that's why I asked, because the changes look like it's regenerated :p

**Comment by damithc** [comment]
@wkurniawan07 can merge this one?

**Pull request merged**